readthedocs/builds/views.py

```diff
@@ -82,6 +83,17 @@ class BuildDetail(BuildBase, DetailView):
     def get_context_data(self, **kwargs):
         context = super(BuildDetail, self).get_context_data(**kwargs)
         context['project'] = self.project
+        issue = "https://github.com/rtfd/readthedocs.org/issues/new?title="
```

Comment:
`urlparse` is a better way to build a URL. There are a number of problems with building by hand through string concatenation.